councilmatic_core/management/commands/data_integrity.py

```diff
@@ -0,0 +1,42 @@
+import pysolr
+import requests
+import logging
+import logging.config
+
+from django.core.management.base import BaseCommand, CommandError
+from django.conf import settings
+
+from councilmatic_core.models import Bill
+
+
+logging.config.dictConfig(settings.LOGGING)
+logging.getLogger("requests").setLevel(logging.WARNING)
+logging.getLogger("pysolr").setLevel(logging.WARNING)
+logger = logging.getLogger(__name__)
+
+class Command(BaseCommand):
+    help = 'Checks for alignment between the Solr index and Councilmatic database'
+
+    def handle(self, *args, **options):
+        councilmatic_count = Bill.objects.all().count()
+
+        try:
+            solr_url = settings.HAYSTACK_CONNECTIONS['default']['URL']
+            requests.get(solr_url)
+        except requests.ConnectionError:
+            message = "ConnectionError: Unable to connect to Solr at {}. Is Solr running?".format(solr_url)
```

Comment:
Should we make this look different from the error that we get from Haystack so that we know where it came from?